### ISSUE TITLE
Not compatible with flexdashboard

### ISSUE THREAD

**Issue by GregorDeCillia** [jbkunst/highcharter#678]
Hello,

I just ran into issues when I tried to use highcharter together with [rstudio/flexdashboard](https://github.com/rstudio/flexdashboard). More specifically, the burger icon in the navigation bar is not working in mobile mode.

![image](https://user-images.githubusercontent.com/9311130/95745248-00eb1980-0c95-11eb-80a7-9c4a11e3d6c2.png)

The problem ist that the jQuery version used in highcharter is not compatible with bootstrap - a css library used by flexdashboard. See the following links for more technical details

* https://www.questionary.wiki/en/questions/software/bootstrap-cannot-convert-object-to-primitive-value
* https://stackoverflow.com/questions/61176484/cannot-convert-object-to-primitive-value-error-in-react-application

Is there any way to downgrade or upgrade the jQuery version when using highcharts? I tried including version 3.5.1 of jQuery via a CDN but then highcharts.js was not loading correctly.

Repex:

    ---
    output: flexdashboard::flex_dashboard
    ---
    
    Info
    =====================================     
    
    ```{r}
    library(highcharter)
    highchart(type = "stock") %>% 
      hc_add_series(AirPassengers)
    ```

**Comment by StefanMusch** [comment]
I actually ran into this issue today too. Is there any update on whether this is fixable? 

@GregorDeCillia did you find a solution to the issue?

**Comment by StefanMusch** [comment]
This issue can be solved through installing the development version of Flexdashboard. See [this issue](https://github.com/rstudio/flexdashboard/issues/328) in the flexdashboard page.

**Comment by GregorDeCillia** [comment]
@StefanMusch Thanks a lot for reporting this fix! I've been using a private fork of highcharter for the last several months as a workaround. Now I'll go back to the dev-version and enjoy all the goodies that were added since 3a0d330d7a2dc775545446439def503968f34be0 :)

Not sure if this upstream fix warrants a close of this issue or if we should wait until flexdashboard is shipped to CRAN again. I'll let @jbkunst decide on this.

**Comment by jbkunst** [comment]
This is solved via updating jquery #722 thanks to @hdrab127
